src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.ts

```diff
@@ -149,12 +149,12 @@ export class MissionRuntimeCreateappStepComponent extends LauncherStep implement
       this.versionId = newVersion.id;
       this.launcherComponent.summary.runtime = runtime;
       this.launcherComponent.summary.runtime.version = newVersion;
-
       // FIXME: use a booster change event listener to do this
```

Comment:
@bartoszmajsak could you use the event to set the mavenArtifact (and remove the FIXME)?

Reply:
Let's make this another issue and PR. WDYT?